internal/driver/mobile/app/android.c

```diff
@@ -85,6 +86,7 @@ void ANativeActivity_onCreate(ANativeActivity *activity, void* savedState, size_
 		hide_keyboard_method = find_static_method(env, current_class, "hideKeyboard", "()V");
 		show_file_open_method = find_static_method(env, current_class, "showFileOpen", "(Ljava/lang/String;)V");
 		show_file_save_method = find_static_method(env, current_class, "showFileSave", "(Ljava/lang/String;Ljava/lang/String;)V");
+		finish_method = find_method(env, current_class, "finishActivity", "()V");
```

Comment:
this should call `finish` instead of `finishActivity`, because it is crashing

Reply:
I don't think so... are you sure that you have built the new fyne command to package the app when testing this code?

Reply:
oh, sorry, my tablet was restarted and now it going, really sorry